### DISCUSSION TITLE
[API Request] Support additional Artist types

### DISCUSSION THREAD

**Discussion by Tolriq** [opensubsonic/open-subsonic-api#24] [API requests]
### Type of change

API tweak

### Proposal description

This is a template discussion to gather all the needs from previous discussions.

- Add support for composers / performers, ... and different roles.
- Add support for multiple artists (See https://github.com/opensubsonic/open-subsonic-api/discussions/8 too)
- Allow proper handling of song artist vs album artists for compilations, ...

### Backward compatibility impact

Should try to be done without compatibility impact.

### Backward compatibility

- [ ] No backward compatibility impact.

### API details

TBD

**Comment by acroyear** [comment]
ah, maybe this is where i should have said this.

I have a need for composer and conductor specifically, for classical music. In addition, I use the 'work' field (or maybe group, can't remember the exact id3 tag) to organize the works on a classical album. E.g., I have a CD of Reich quartets - the album is "String Quartets", but within i have the groups of "Different Trains" "Triple Quartet", etc.

Of course, having this kind of data also means there should be search endpoints to find it. maybe that scales things up a bit too much.

**Comment by Tolriq** [comment]
See proposal https://github.com/opensubsonic/open-subsonic-api/discussions/52
